src/Tizen.NUI.Components/Controls/Slider.cs

```diff
@@ -21,6 +21,32 @@
 
 namespace Tizen.NUI.Components
 {
+    /// <summary>
+    /// Slider value changed event data.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public class SliderValueChangedEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Curren value
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public float CurrentValue { get; set; }
+    }
+
+    /// <summary>
+    /// Slider sliding finished event data.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public class SliderSlidingFinishedEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Curren value
```

Comment:
What does this value represents? The sliding distance or current position in the X axis?
```suggestion
        /// Current value
```

Reply:
Thank you for your comment.
The current value is not the sliding distance and the current position is not the exact explanation for the current value.
So I write it as "Current Slider value" because slider uses the term "current value" already.